apps/next-no-cache/next.config.mjs

```diff
@@ -1,9 +1,19 @@
+import path from 'node:path'
 import withBundleAnalyzer from '@next/bundle-analyzer'
+import sanityPkg from 'sanity/package.json' assert {type: 'json'}
 
 function requireResolve(id) {
   return import.meta.resolve(id).replace('file://', '')
 }
 
+const sanityExports = {}
+for (const key of Object.keys(sanityPkg.exports)) {
+  if (key === '.') continue
+  const subexport = path.join('sanity', key)
+  sanityExports[subexport] = requireResolve(subexport)
+}
+sanityExports['sanity'] = requireResolve('sanity')
+
```

Comment:
We'll ship a first class utility for this in `next-sanity` later. This feature is based on the new aliasing that `sanity dev` does since v3.53.0

Reply:
Cool! Also TIL about Import Assertions